src/application.ts

```diff
@@ -78,6 +79,7 @@ export class PocketGatewayApplication extends BootMixin(ServiceMixin(RepositoryM
     const relayRetries: string = POCKET_RELAY_RETRIES || ''
     const databaseEncryptionKey: string = DATABASE_ENCRYPTION_KEY || ''
     const defaultSyncAllowance: number = parseInt(DEFAULT_SYNC_ALLOWANCE) || -1
+    const defaultLogLimitBlocks: number = parseInt(LOG_LIMIT_BLOCKS) || -1
```

Comment:
if the const is 'defaultLogLimitBlocks' then please make the ENV:

DEFAULT_LOG_LIMIT_BLOCKS